tests/testthat/cubes/mr-by-cat-profiles-stats.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
See slack discussion; I think this fixture needs to be sanitized.

Reply:
I think we can remove it from this repo